andino_gz/launch/andino_gz.launch.py

```diff
@@ -52,10 +53,10 @@ def generate_launch_description():
     params_file = LaunchConfiguration('params_file')
 
     # Obtains world path.
-    world_path = PathJoinSubstitution([pkg_andino_gz, 'worlds', world_name])
+    world_path = PathJoinSubstitution([gz_resource_path, 'worlds', world_name])
     log_world_path = LogInfo(msg=TextJoin(substitutions=["World path: ", world_path]))
     # Obtains the map path.
-    map_path = PathJoinSubstitution([pkg_andino_gz, 'maps', map_name, TextJoin(substitutions=[map_name ,'.yaml'])])
+    map_path = PathJoinSubstitution([gz_resource_path, 'maps', map_name, TextJoin(substitutions=[map_name ,'.yaml'])])
```

Comment:
- It would be great if we can support both, if the map is not found in "internal" path then it will check another path.
 - I am not sure if `GZ_RESOURCE_PATH` is the best as it is too similar to `GZ_SIM_RESOURCE_PATH`. I'd prefer we use something like `ANDINO_GZ_RESOURCE_PATH`